src/server/scripts/EasternKingdoms/OperationGnomeregan/operation_gnomeregan.cpp

```diff
@@ -0,0 +1,2567 @@
+                        AddWaypoint(3, -5055.729004f, 735.350525f, 260.556061f);
+                        AddWaypoint(4, -5054.227051f, 735.254822f, 261.244995f);
+                        AddWaypoint(5, -5046.759766f, 735.879211f, 256.475433f);
+                        AddWaypoint(6, -5035.186523f, 739.123962f, 256.475433f);
+                        AddWaypoint(7, -4974.740234f, 734.052185f, 256.263763f);
+                        AddWaypoint(8, -4963.679688f, 732.872192f, 257.911133f);
+                        AddWaypoint(9, -4959.834961f, 734.894470f, 259.341797f);
+                        AddWaypoint(10, -4963.027832f, 735.367432f, 258.783722f);
+                        break;
+                    case 17:
+                        AddWaypoint(0, -5104.795898f, 717.425720f, 257.537598f);
+                        AddWaypoint(1, -5095.783203f, 716.306946f, 260.370605f);
+                        AddWaypoint(2, -5085.620117f, 716.721130f, 260.557373f);
+                        AddWaypoint(3, -5056.110352f, 727.733398f, 260.561371f);
+                        AddWaypoint(4, -5054.630371f, 727.904541f, 261.246185f);
+                        AddWaypoint(5, -5047.102051f, 731.708801f, 256.475403f);
+                        AddWaypoint(6, -5036.331055f, 730.962219f, 256.475403f);
+                        AddWaypoint(7, -4974.770996f, 726.103333f, 256.258118f);
+                        AddWaypoint(8, -4948.733398f, 723.634705f, 260.412781f);
+                        AddWaypoint(9, -4947.263184f, 723.494263f, 261.646088f);
+                        AddWaypoint(10, -4940.096191f, 717.815491f, 261.646088f);
+                        AddWaypoint(11, -4939.854980f, 712.329895f, 261.644684f);
+                        AddWaypoint(12, -4940.705566f, 713.400269f, 261.644684f);
+                        break;
+                    case 18:
+                        AddWaypoint(0, -5109.956543f, 717.818481f, 255.921844f);
+                        AddWaypoint(1, -5095.956055f, 716.377014f, 260.314667f);
+                        AddWaypoint(2, -5085.541992f, 716.675903f, 260.557373f);
+                        AddWaypoint(3, -5056.114258f, 727.704346f, 260.561310f);
+                        AddWaypoint(4, -5054.369629f, 727.836121f, 261.243469f);
+                        AddWaypoint(5, -5047.054199f, 731.594299f, 256.475586f);
+                        AddWaypoint(6, -5036.313477f, 731.290100f, 256.475586f);
+                        AddWaypoint(7, -4975.180664f, 725.747314f, 256.261780f);
+                        AddWaypoint(8, -4964.648926f, 725.515686f, 257.849091f);
+                        AddWaypoint(9, -4960.842285f, 722.796204f, 259.332458f);
+                        AddWaypoint(10, -4964.441895f, 722.924133f, 258.709137f);
+                        break;
+                    default:
+                        sLog->outError("Unexpected movement variation (%i) in npc_og_infantryAI::SetupMovement call!", variation);
```

Comment:
this is needed or only used for your debugging?

Reply:
applies to all of them in the file

Reply:
?? that looks useful, what u talking about??

Reply:
used for debugging and needed, because it shouldn't occur